environment/src/main/java/com/sequenceiq/environment/environment/domain/EnvironmentPlatform.java

```diff
@@ -0,0 +1,8 @@
+package com.sequenceiq.environment.environment.domain;
+
+public enum EnvironmentPlatform {
+    // everything is PaaS
+    PAAS,
+    // SDX SaaS, others PaaS
```

Comment:
might not be granular enough in the future

Reply:
yes, I discussed it with @lnardai , I'll remove this completely, not needed